### PR TITLE
Updating README

### PR DESCRIPTION
**Pull request by shreypuranik**
Updating README with a small note on how the security advisories project can be used in build plans.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,8 @@ applications beside manual checks:
 
         curl -H "Accept: application/json" https://security.symfony.com/check_lock -F lock=@/path/to/composer.lock
 
+It is convenient to include the call to the Symfony Web Service in any build plans or existing Continuous Integration workflow, so that you can check for any new unaddressed vulnerabilities every time your application is built. Simply include the curl command (from above) in any build.sh (or equivalent) file (pointng to the local composer.lock).
```
Comment (stof) [inline README.md]: typo: `pointing` (missing `i`)

**Comment by stof** [comment]
Based on the work happening in https://github.com/symfony/diversity/issues/9, I suggest avoiding the word `simply` in the new note.

File: README.md [hunk 2]
```diff
@@ -34,6 +34,8 @@ applications beside manual checks:
 
         curl -H "Accept: application/json" https://security.symfony.com/check_lock -F lock=@/path/to/composer.lock
 
+It is convenient to include the call to the Symfony Web Service in any build plans or existing Continuous Integration workflow, so that you can check for any new unaddressed vulnerabilities every time your application is built. This can be achieved by including the curl command (from above) in any build.sh (or equivalent) file (pointing to the local composer.lock).
```
Comment (BackEndTea) [inline README.md]: I would press this more urgent and say something along the lines of 
"Its advised to include..."

Comment (BackEndTea) [reply README.md]: And maybe refer to https://github.com/sensiolabs/security-checker as well, as an alternative to a barebones curl.

Comment (shreypuranik) [reply README.md]: Thanks for your feedback @BackendTea. I'll update the Pull Request taking these comments into account.

Comment (shreypuranik) [reply README.md]: Hi @BackEndTea , I have updated the Pull Request. Hope you can review these updates whenever you have a chance to do so.

File: README.md [hunk 3]
```diff
@@ -34,6 +34,10 @@ applications beside manual checks:
 
         curl -H "Accept: application/json" https://security.symfony.com/check_lock -F lock=@/path/to/composer.lock
 
+It is advised to include the call to the Symfony Web Service in any build plans or existing Continuous Integration workflow, so that you can check for any new unaddressed vulnerabilities every time your application is built. This can be achieved by including the curl command (from above) in any build.sh (or equivalent) file (pointing to the local composer.lock). 
+
+An alternative to using curl in your build plans or Continuous Integration workflow is to use the Sensiolabs Security Checker dependency which you can install as a dependency on it's own. More details are available at https://github.com/sensiolabs/security-checker
```
Comment (cebe) [inline README.md]: small formatting improvement:

```suggestion
An alternative to using curl in your build plans or Continuous Integration workflow is to use the Sensiolabs Security Checker dependency which you can install as a dependency on it's own. More details are available at <https://github.com/sensiolabs/security-checker>.
```

File: README.md [hunk 4]
```diff
@@ -34,6 +34,10 @@ applications beside manual checks:
 
         curl -H "Accept: application/json" https://security.symfony.com/check_lock -F lock=@/path/to/composer.lock
 
+It is advised to include the call to the Symfony Web Service in any build plans or existing Continuous Integration workflow, so that you can check for any new unaddressed vulnerabilities every time your application is built. This can be achieved by including the curl command (from above) in any build.sh (or equivalent) file (pointing to the local composer.lock). 
+
+An alternative to using curl in your build plans or Continuous Integration workflow is to use the Sensiolabs Security Checker dependency which you can install as a dependency on it's own. More details are available at <https://github.com/sensiolabs/security-checker>.
```
Comment (ntzm) [inline README.md]:
```suggestion
An alternative to using curl in your build plans or Continuous Integration workflow is to use the Sensiolabs Security Checker dependency which you can install as a dependency on its own. More details are available at <https://github.com/sensiolabs/security-checker>.
```

**Comment by fabpot** [comment]
Closing in favor of #405

**Pull request closed without merging**